xarray/core/common.py

```diff
@@ -1172,6 +1243,57 @@ def full_like(other, fill_value, dtype: DTypeLike = None):
         filled with fill_value. Coords will be copied from other.
         If other is based on dask, the new one will be as well, and will be
         split in the same chunks.
+
+    Examples
+    --------
+
+    >>> import numpy as np
+    >>> import xarray as xr
+    >>> x = xr.DataArray(np.arange(6).reshape(2, 3),
+    ...                  dims=['lat', 'lon'],
+    ...                  coords={'lat': [1, 2], 'lon': [0, 1, 2]})
+    >>> x
+    <xarray.DataArray (lat: 2, lon: 3)>
+    array([[0, 1, 2],
+           [3, 4, 5]])
+    Coordinates:
+    * lat      (lat) int64 1 2
+    * lon      (lon) int64 0 1 2
+    >>> xr.full_like(x, 1)
```

Comment:
To the extent you have the patience, I thought the style above with a line break before the long expressions was v nice to read